cmd/collectors/ems/ems.go

```diff
@@ -652,7 +643,16 @@ func (e *Ems) HandleResults(result []gjson.Result, prop map[string][]*emsProp) (
 			count += instanceLabelCount
 		}
 	}
-	return m, count
+
```

Comment:
Can we calculate `instanceCount` in the above loops above, so we don't have to loop through all instances of all matrices again here?

Reply:
It's added here mainly for `autoresolve` case.
In that case, we haven't get any records from ONTAP, so it won't run the above loop, but still we want to export 1 instance which would be resolved based on time.